crates/typst/src/foundations/dict.rs

```diff
@@ -156,6 +158,29 @@ impl Dict {
 
 #[scope]
 impl Dict {
+    /// Converts a value into a dictionary
+    ///
+    /// This function can convert a module into a dictionary
+    ///
+    /// ```example
+    /// #include "test.typ" as test
```

Comment:
This example fails to compile because (a) `import` should be used rather than `include` and (b) `test.typ` doesn't exist. Since there are no files present that could be imported, maybe we can just use one of the standard library modules. E.g. `#dictionary(sys).at("version")`.